src/ralph/models/xapi/navigation/events.py

```diff
@@ -0,0 +1,33 @@
+"""Navigation xAPI event definitions"""
```

Comment:
For semantic consistency, this file should rather be called `statements.py`, right?